_projects/git_task_list.md

```diff
@@ -0,0 +1,87 @@
+---
+collaborating_projects:
+  - coala
+  - meta-review
+desc: "List pull request/merge request (in gitlab) that have no reviewer yet. Provide an easy way for new comer / developer to do contribution to a project by giving review to other people pull requests."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - https://github.com/coala/projects/issues/480
+  - https://github.com/coala/coala/issues/3615
+  - https://github.com/coala/coala/issues/4080
+markdown: git_task_list.md
+mentors:
+  - jayvdb
```

Comment:
I am not on your list of mentors, and the first mentor listed here should be your primary mentor.